.github/workflows/schedule.cli.yml

```diff
@@ -0,0 +1,38 @@
+name: Schedule cli
+on:
+  workflow_run:
+    workflows: ["Pre submits cli"]
+    types: [completed]
+    branches: [main]
```

Comment:
again, we probably want this to run on `release/*` branches.

Reply:
I've removed it since it will be triggered by the other workflow.